src/common/header/Header.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Nav, Navbar } from 'react-bootstrap';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const Header = (props: { children: React.ReactNode }) => (
+  <Navbar bg="primary" variant="dark">
+    <Navbar.Brand>{props.children}</Navbar.Brand>
+    <Nav className="mr-auto"></Nav>
+  </Navbar>
+);
+
+export default Header;
```

Comment:
Same here, this component has its own Jira ticket, we can implement it in a different PR.

Reply:
I'll make another PR for this component.